src/mobile-services.json

```diff
@@ -27,6 +27,23 @@
       "type": "metrics",
       "url": "https://www.mocky.io/v2/5aa696133100001335e716e0",
       "config": {}
+    },
+    {
+      "id": "push",
+      "name": "push",
+      "type": "push",
+      "url": "https://ups-dm-myproject-1.193b.starter-ca-central-1.openshiftapps.com/",
+      "config": {
+        "ios": {
+          "variantId": "f85015b4-a762-49a7-a36f-34a451f819a4",
+          "variantSecret": "978b35d6-7058-43b4-8c37-4dc30022ebda"
+        },
+        "android": {
+          "variantId": "795d5953-558c-4026-9407-194169fbf7fa",
+          "variantSecret": "92fd41dd-505b-4513-aaea-d02e27251695",
+          "senderId": "754329551899"
```

Comment:
We are not currently using that field and thigs seems to work. I have bug in progress to check why previos version of plugin needed that.